lazer/sdk/rust/protocol/src/message.rs

```diff
@@ -15,7 +77,7 @@ pub struct EvmMessage {
 
 impl EvmMessage {
     pub fn serialize(&self, mut writer: impl Write) -> anyhow::Result<()> {
-        writer.write_u32::<BE>(EVM_FORMAT_MAGIC)?;
+        writer.write_u32::<LE>(EVM_FORMAT_MAGIC)?;
```

Comment:
isn't this a breaking change? How does the current contracts in EVM that expect BE work with this?

Reply:
It's not a breaking change because I changed the constant value.